rules/do-you-know-the-best-practices-for-frontmatter-in-markdown/rule.md

```diff
@@ -17,41 +19,37 @@ However, when not structured properly, it can lead to parsing errors, inconsiste
 
 <!--endintro-->
 
-#### Key-Value Pairs
+#### Key-Value Pair Organization
 
-Frontmatter in markdown primarily utilizes key-value pairs to represent metadata. Each key represents an attribute, like the title or the author, and the value associated with it provides specific information related to that attribute.
+Frontmatter is metadata serialized into a plain text format primarily yaml but can also be toml, or json. In Frontmatter, each key represents an attribute, like the title or the author, and the value associated with it provides specific information related to that attribute.
 
 - **Keys** are always strings and should be descriptive enough to indicate the type of data they hold.
 - **Values** can be strings, numbers, dates, or even arrays, depending on the data you're representing.
 
 Using key-value pairs ensures a standardized format, which in turn makes it easier for both humans and machines to read and interpret the data. Moreover, this structured approach ensures that markdown processors can reliably extract and utilize the metadata, whether it's for rendering a webpage title, categorizing posts, or any other function.
 However, avoid non-standard practices like mixing data types or adding unnecessary complexity:
 
-::: greybox
 ```
 –––
 title+author: My Article by John
 2023-10-31
 –––
 ```
-:::
 
 ::: bad 
-Figure: Bad Example {Non-standard practices can lead to parsing errors and inconsistencies.} 
+❌Figure: Bad Example of non-standard practices can lead to parsing errors and inconsistencies.
 :::
 
-::: greybox
 ```
 –––
 title: My Article
-author: John Doe
+author: Bob Northwind
 date: 2023-10-31
 –––
 ```
-:::
 
 ::: good 
-Figure: Good Example {Clear key-value pairs make it easy to understand and extract the metadata.} 
+✅Figure: Good Example of clear key-value pairs make it easy to understand and extract the metadata.
```

Comment:
```suggestion
Figure: Good Example - key-value pairs make it easy to understand and extract the metadata.
```

bad/good examples have classes that apply ticks/crosses automatically

Reply:
I will fix it now